components/chef-ui-library/src/molecules/chef-toolbar/chef-toolbar.scss

```diff
@@ -1,10 +1,5 @@
 chef-toolbar {
   chef-button {
-    margin-top: 0;
-
-    &:first-child {
```

Comment:
the way things are setup right now all buttons in the toolbar are first childs so this was being applied to all the buttons anyway